web/src/compose_validate.ts

```diff
@@ -717,18 +717,19 @@ function validate_private_message(): boolean {
     return true;
 }
 
-export function check_overflow_text(): number {
+export let check_overflow_text = ($elem: JQuery): number => {
```

Comment:
`$container` might be a better name than `$elem` for this, to indicate what its role is in the code here.